libs/pages/elewa/home/src/lib/features-elewa-home.module.ts

```diff
@@ -23,7 +24,8 @@ import { HomeRoutingModule } from './home.routing';
     LayoutModule,
     NgOptimizedImage,
     CardsModule,
-    HomeRoutingModule
+    HomeRoutingModule,
+    SocialImpactModule
   ],
```

Comment:
Not allowed.
Pages cannot import other page modules, use shared modules instead. May cause circular dependency.